soc/riscv/riscv-privilege/telink_w91/compiler.h

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
This file seems like duplication of standard zephyr functionality
See: https://docs.zephyrproject.org/apidoc/latest/gcc_8h_source.html

Reply:
removed